tmt/convert.py

```diff
@@ -448,10 +448,11 @@ def read(
             restraint_file = True
             echo(style('Restraint ', fg='blue'), nl=False)
 
-    if filename is None:
+    if filename is None and not polarion:
         raise GeneralError('filename is not defined')
```

Comment:
Is the error message still valid? If code gets to this `raise`, it must mean `polarion` is also not set, would that be worth a mention?